packages/adapters-library/src/adapters/compound-v2/tests/snapshots/ethereum.positions.supply.json

```diff
@@ -1,6 +1,6 @@
 {
   "blockNumber": 19383091,
-  "aggregatedValues": ["USD7,575.19", "USD116,358.72", "USD64,954.14"],
+  "aggregatedValues": ["USD7,575.19", "USD116,358.67", "USD64,954.14"],
```

Comment:
im not sure what could have caused this difference but dont think its worth spending time on unless it happens again